src/lib.rs

```diff
@@ -251,6 +249,7 @@ mod wrapper {
     pub mod strings;
 
     /// Actual communication with the JVM
+    #[allow(clippy::unit_arg)]
```

Comment:
(Here and elsewhere) I'd add a short comment explaining why it is allowed if it is not apparent.